content/documentation/install-uninstall.md

```diff
@@ -26,7 +26,20 @@ The Quire installation process may take a minute or two, during which time there
 
 
 {{< q-class "box warning" >}}
-- The Quire installation process may take a minute or two, during which time there will be messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. If you receive an error message or get a "command not found" message after entering `quire --help` in step 4., visit our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
+- The Quire installation process may take a minute or two, during which time there will be messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR.
```

Comment:
Yes, this feels like a lot to stuff into a warning box. It would also be relevant to the full install below. 

What about having the warning box be just the first paragraph as it was, with something added like "If you see error messages that say "permission denied", see the section below."? And then link to a new sub-section under the full install instructions that has these instructions for permissions errors.

Reply:
OK I just made this adjustment. Let me know if the new version works.